dronuts-backend/src/endpoints/order_routes.js

```diff
@@ -1,26 +1,25 @@
 const {startDatabase} = require('../db/db');
 const {default: order_schema} = require('../schema/order_schema');
+const {persistOrderStatus} = require('./order_status_routes');
 
 async function persistOrder(requestBody) {
     const client = startDatabase();
-
     client.connect(async function(err, client) {
         const collection = client.db('dronuts').collection('Order');
         const num_orders = await collection.countDocuments();
-        requestBody.id = num_orders + 1;
+        const orderId = num_orders + 1;
+        requestBody.id = orderId;
         try {
             const new_order = new order_schema(requestBody);
             new_order.validate();
-
             collection.insertOne(new_order, function() {
                 client.close();
             });
+            persistOrderStatus(orderId);
         } catch (error) {
-            console.log(error);
-            return false;
+            throw err;
```

Comment:
Is there a way we can keep a log of errors so we can keep track of things that go wrong in the backend?

Reply:
Actually, yeah. I should keep the logging in.